app/views/events/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= link_to "Back to Event page", events_path %><br>
+<div>
+<p><strong>Event Title:</strong><%= @event.title %></p>
+</div>
+<div>
+<p><strong>Description:</strong><%= @event.description %></p>
+</div>
+<div>
+<p><strong>Date:</strong><%= @event.date_of_event %></p>
+
+</div>
+<div>
+ <%= button_to "Attend", attended_event_event_path(@event), method: :post, confirm: "really?" %>
+</div>
+<div>
+<% if @event.attendees.empty? %>
+    <%= "No user attended this event" %>   
+<% else %>
+<p> List of attendee users:</p>
+<% @event.attendees.each do |attendee| %>
+<ul>
+<li>
+<%= attendee.name %>
+</li>
+</ul>
+	<% end %>
```

Comment:
**[OPTIONAL]**
There should be absolutely NO logic in views. Move it to a helper file.
**Don't know where to start or how to?** See  [Rails View Helpers Best Practices](https://medium.com/@acushla4real/rails-view-helpers-best-practices-guide-d7d7b6db3db4)